btree/btree.c

```diff
@@ -1015,34 +1135,51 @@ static bool node_shift_is_valid(int shift)
 	return shift >= NODE_SHIFT_MIN && shift < NODE_SHIFT_MIN + 0x10;
 }
 
-static bool segaddr_is_valid(const struct segaddr *addr)
+/**
+ * This function should be called right after the allocation to make sure that
+ * the allocated memory starts at a properly aligned address.
+ *
+ * @author 530902 (06-Apr-21)
```

Comment:
This should be something more human readable. :-) But in general, authorship is recorded in the git logs, there is no need to duplicate it in the source.